src/TreeHelper/RobotOutOfFieldHelper.cpp

```diff
@@ -0,0 +1,32 @@
+//
+// Created by jessevw on 19.11.19.
+//
+#include <include/roboteam_ai/TreeHelper/RobotOutOfFieldHelper.h>
+#include <include/roboteam_ai/bt/composites/Sequence.hpp>
+#include <include/roboteam_ai/conditions/RobotOutside.h>
+#include <include/roboteam_ai/skills/gotopos/GTPSpecial.h>
+#include "bt/RoleDivider.h"
+#include "bt/Role.h"
+#include "skills/Attack.h"
+
+// Reference wrapper
+
+namespace bt {
+    RobotOutOfFieldHelper::RobotOutOfFieldHelper() {
+
+    }
+    std::shared_ptr<bt::Node> RobotOutOfFieldHelper::createRobotOutOfFieldHelper() {
+        auto localbb = std::make_shared<bt::Blackboard>();
+        auto robotOutsideBB = std::make_shared<bt::Blackboard>();
+
+        robotOutsideBB->setString("type", "defaultType");
```

Comment:
just a random comment, i do not like blackboard, it's runtime and slow, if you really need variables with different names that aren't known at compiletime, then sure, but things like these string literals can just be regular members

Reply:
Agreee! :dango: